lang/ilias_de.lang

```diff
@@ -5260,6 +5261,7 @@ common#:#pd_items_news#:#Einschließlich Neuigkeiten der ausgewählten Objekte
 common#:#pdf_export#:#PDF-Export
 common#:#perm_settings#:#Rechte
 common#:#perma_link#:#Link zu dieser Seite
+common#:#perma_link_copied#:#Link wurde in die Zwischenablage kopiert.
```

Comment:
Here we have more space for information: 'Link zu dieser Seite wurde in die Zwischenablage kopiert.'